fabtests/scripts/runfabtests.sh

```diff
@@ -532,7 +532,7 @@ function multinode_test {
 	s_cmd="${BIN_PATH}${test_exe} ${S_ARGS} -s ${S_INTERFACE}"
 	${SERVER_CMD} "${EXPORT_ENV} $s_cmd" &> $s_outp &
 	s_pid=$!
-	sleep 5
+	sleep 1
```

Comment:
This change reverts the change from the other commit?

Reply:
Yeah this was a guess attempt at a fix and I guess I undid the change incorrectly in git. I'll see if I can change it so that this non-change doesn't show up in the log at all